src/cascadia/TerminalApp/SnippetsPaneContent.xaml

```diff
@@ -0,0 +1,262 @@
+                                </ResourceDictionary>
+                            </Button.Resources>
+                        </Button>
+
+                        <local:HighlightedTextControl Grid.Column="1"
+                                                      HorizontalAlignment="Left"
+                                                      IsTabStop="False"
+                                                      Text="{x:Bind HighlightedName, Mode=OneWay}" />
+
+                        <!--
+                            BODGY: I can't choose different templates if this item is nested or not.
+                            (See https://github.com/microsoft/microsoft-ui-xaml/issues/2121)
+                            But what I can do, is toss the text into a 0-height row to hide it.
+                        -->
+                        <TextBlock Grid.Row="{x:Bind Row, Mode=OneWay}"
+                                   Grid.Column="1"
+                                   Margin="12,0,12,6"
+                                   FontFamily="Cascadia Mono, Consolas"
+                                   IsTextSelectionEnabled="False"
+                                   Style="{ThemeResource BodyTextBlockStyle}"
+                                   Text="{x:Bind Input}"
+                                   Visibility="{Binding ElementName=rootItem, Path=IsSelected}" />
+                    </Grid>
+                </mux:TreeViewItem>
+            </DataTemplate>
+
+            <ResourceDictionary.ThemeDictionaries>
+                <!--  same as in MainPage, this is SolidBackgroundFillColorTertiary  -->
+                <ResourceDictionary x:Key="Dark">
+                    <Color x:Key="PageBackground">#282828</Color>
+                    <Color x:Key="PlayButtonHoveredColor">#90ef90</Color>
+                    <Color x:Key="PlayButtonNormalColor">#8888</Color>
+                </ResourceDictionary>
+                <ResourceDictionary x:Key="Light">
+                    <Color x:Key="PageBackground">#F9F9F9</Color>
+                    <Color x:Key="PlayButtonHoveredColor">#257f01</Color>
+                    <Color x:Key="PlayButtonNormalColor">#88222222</Color>
+                </ResourceDictionary>
+                <ResourceDictionary x:Key="HighContrast">
+                    <!--  Define resources for HighContrast mode here  -->
+                    <StaticResource x:Key="PageBackground"
+                                    ResourceKey="SystemColorWindowColorBrush" />
+                </ResourceDictionary>
+
+            </ResourceDictionary.ThemeDictionaries>
+
+        </ResourceDictionary>
+    </UserControl.Resources>
+
+    <!--  Don't allow the user to focus the root pane, but do allow individual controls to be focused  -->
+
+    <Grid AllowFocusOnInteraction="False"
+          AutomationProperties.AccessibilityView="Raw"
+          Background="{ThemeResource PageBackground}"
+          TabFocusNavigation="Cycle">

```

Comment:
```suggestion
          Background="{ThemeResource PageBackground}">
```
We might not need this one since `TabFocusNavigation` is set to `Cycle` in the top-level UserControl. I suggest taking this one out. Minorly concerned if we add another control in the future (and it's not in the grid), we may be stuck in the cycle in this grid

Reply:
Actually, this one was the load bearing one, not the other. Added comment.